### DISCUSSION TITLE
Add a keyboard shortcut for Run / View Report / As PDF

### DISCUSSION THREAD

**Discussion by hvbtup** [eclipse-birt/birt#821] [Ideas]
While developing a report, very frequently one needs to check the report rendered as PDF. This is quite cumbersome to do with the mouse. A keyboard shortcut for this is something which report developers often ask for. It seems impossible to do this with "customize perspective"

**Comment by ruspl-afed** [comment]
What keyboard combination do you have in mind? We should avoid conflicts with existing combinations for both IDE and RCP scenario

**Reply by hvbtup** [reply]
Personally, I think that Ctrl+F11 or Ctrl+R are natural choices.

Could be even more useful if the IDE remembered my last choice of the format (As PDF, As DOC, As HTML, ...) and repeated that action if available.

I only now found the Ctrl+Shift+L menu, but I didn't findy "View Report... or "View Report as PDF" anywhere there.
